src/groups/mqb/mqbs/mqbs_filebackedstorage.cpp

```diff
@@ -112,25 +112,24 @@ FileBackedStorage::FileBackedStorage(
 , d_store_p(dataStore)
 , d_queueKey(queueKey)
 , d_config()
-, d_queueUri(queueUri, allocator)
+, d_queueUri(queueUri, d_allocator_p)
```

Comment:
We assume that `d_allocator_p` field is already initialized and this allocator is checked. `allocator` arg passed to constructor is not yet checked and it's not good to use it when we have `d_allocator_p`